website/content/docs/configuration/consul.mdx

```diff
@@ -334,6 +336,58 @@ consul {
 }
 ```
 
+### Consul ACL policy for Nomad
+
+Nomad agents need access to Consul in order to register themselves in the
+service catalog and discover other Nomad agents via service discovery for
+automatic clustering. Nomad clients use Consul tokens from Workload Identity to
+register services and checks but need permissions on their own token to
+deregister. Nomad servers also create [configuration
+entries][consul_config_entry] for Consul Service Mesh, so the specific
```

Comment:
[configuration entries][consul_config_entry] -> link is broken - need to add the URL entry  

```[consul_config_entry]: /consul/docs/connect/config-entries```